README.md

```diff
@@ -3,33 +3,71 @@
 __URLs__
 
 * Name all of the parts of the url that you can remember.  In your own words describe what they do.
+
+```Protocol - either http or https. Just establishes which protocol is being used.
+```
+```Domain - a name that directs to the correct server address.
+```
+```Port - a specific program reference.
+```
+```Path - the specific folder location being directed to.
+```
+```Query - (optional) if there is some input on the page that needs to be duplicated. Such as a search.
+```
+```Link - a specific link on the HTML to direct to.
+```
+
 * Name the pieces of the following urls:
-	* `https://www.google.com/`
-	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367`
-	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy`
-	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error`
+	* `https://www.google.com/` ```protocol/domain```
+	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367` ```protocol/domain/path```
+	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy` ```protocol/domain/port/path/query/link```
+	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error` ```protocol/domain/path/link```
 * Can a server use more than 1 port?
+```I don't believe so
```

Comment:
This question is not that great.

Technically speaking a server can use more than 1 port - but I don't think the person who wrote these questions was thinking about that.